tests/QARegressionTests/test_Issues/test_CoreIssue145.py

```diff
@@ -0,0 +1,74 @@
+import pandas

```

Comment:
I don't think we need to commit tests for the business operators. Those are handled in that repo

Reply:
removed